### ISSUE TITLE
-Wcompare-distinct-pointer-types in drivers/nvmem/jz4780-efuse.c

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#922]
Reported-by: kernelci.org bot <bot@kernelci.org>

```
    drivers/nvmem/jz4780-efuse.c:76:24: warning: comparison of distinct pointer types ('typeof (bytes) *' (aka 'unsigned long *') and 'typeof ((start + 32) - offset) *' (aka 'unsigned int *')) [-Wcompare-distinct-pointer-types]
```

**Comment by nathanchance** [comment]
Patch independently submitted: https://lore.kernel.org/lkml/79e1dec195d287001515600b1dae0bcaa33fbf65.1583522277.git.hns@goldelico.com/

**Comment by nathanchance** [comment]
Fixed in next-20200310.
